modules/kafka/src/main/java/org.testcontainers.containers/KafkaContainer.java

```diff
@@ -0,0 +1,78 @@
+package org.testcontainers.containers;
+
+import org.testcontainers.utility.Base58;
+
+import java.util.stream.Stream;
+
+public class KafkaContainer extends GenericContainer<KafkaContainer> {
+
+    public static final int KAFKA_PORT = 9092;
+
+    public static final int ZOOKEEPER_PORT = 2181;
+
+    protected String externalZookeeperConnect = null;
+
+    protected SocatContainer proxy;
+
+    public KafkaContainer() {
+        this("4.0.0");
+    }
+
+    public KafkaContainer(String confluencePlatformVersion) {
```

Comment:
AFAIK it's `confluent`, not `confluence`.

WDYT about allowing non-confluent images? We are i.e. using https://hub.docker.com/r/ches/kafka/ for our acceptance-tests. This image is missing `KAFKA_INTER_BROKER_LISTENER_NAME`, but it should be trivial to update the image accordingly, the other ENV params should be equal.

Reply:
> AFAIK it's confluent, not confluence.

👍 😄 

> WDYT about allowing non-confluent images
We add too much customization, if one wants to use his custom image, I would rather recommend him to build it on top of GenericContainer